src/citrine/resources/audit_info.py

```diff
@@ -0,0 +1,49 @@
+from typing import Optional
+import arrow
+
+
+class AuditInfo:
+    """
+    Model that holds audit metadata. AuditInfo objects should not be created by the user.
+
+    Parameters
+    ----------
+    created_by: str
+        ID of the user who created the object
+    created_at: int
+        Time, in ms since epoch, at which the object was created
+    updated_by: Optional[str]
+        ID of the user who most recently updated the object
+    updated_at: Optional[int]
+        Time, in ms since epoch, at which the object was most recently updated
+
+    """
+
+    def __init__(self, created_by: str, created_at: int,
```

Comment:
Can you use rich types here (UUID and dates)? Something similar to https://github.com/CitrineInformatics/citrine-python/blob/develop/src/citrine/resources/dataset.py#L73 maybe?

Reply:
Can we assume that `created_by` and `updated_by` will always be uuid4? If so, then yes, I think this is a good idea

Reply:
Yeah they will always be those types

Reply:
Changed to be UUID and Datetime